.vscode/settings.json

```diff
@@ -8,5 +8,10 @@
 		}
 	],
 	"cmake.configureOnEdit": false,
-	"cmake.configureOnOpen": false
+	"cmake.configureOnOpen": false,
+	"files.associations": {
+		"": "",
+		"hw_can.h": "c",
+		"app_imd.h": "c"
+	}
```

Comment:
Can you revert
(I have a PR to delete this file from GitHub so this'll stop happening, sorry!)